mycity/mycity/intents/trash_intent.py

```diff
@@ -174,21 +181,20 @@ def validate_found_address(found_address, user_provided_address):
 
     # Allow for mismatched "Road" street_type between user input and ReCollect API
     if "rd" in found_address["street_type"].lower() and \
-        "road" in user_provided_address["street_type"].lower():
+            "road" in user_provided_address["street_type"].lower():

```

Comment:
These spaces intentional? Is that PEP8 style?

Reply:
I believe that is PEP8 related, was from `⌘+⌥+L` in PyCharm.